mono/metadata/object-internals.h

```diff
@@ -1807,18 +1813,27 @@ mono_error_set_pending_exception (MonoError *error);
 MonoArray *
 mono_glist_to_array (GList *list, MonoClass *eclass, MonoError *error);
 
+MonoArrayHandle
+mono_glist_to_array_handle (GList *list, MonoClass *eclass, MonoError *error);
```

Comment:
I don't think you define `mono_glist_to_array_handle` anywhere in this PR.  (also it seems like it would be difficult to implement without either: (a) holding raw pointers in the glist, or (b) holding a potentially unbounded number of handles in the glist.)

Reply:
Right, some of them are just declarations. Should I remove all just-declarations? There are a few others, like all but one write barrier. I was first really ambitious was going to declare a bunch and implement them all, but later did a more as-needed/used approach.

Reply:
Please only declare functions that are defined in the PR.  (On a personal note: linker errors always make me think I did something wrong - I don't usually expect the called function to be missing)

Reply:
Ok I'll fix that